functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import pypyodbc
+import os
+import azure.functions as func
+import json
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    db_server = os.environ["ENV_DATABASE_SERVER"]
+    db_name = os.environ["ENV_DATABASE_NAME"]
+    db_username = os.environ["ENV_DATABASE_USERNAME"]
+    db_password = os.environ["ENV_DATABASE_PASSWORD"]
+    driver = '{ODBC Driver 17 for SQL Server}'

```

Comment:
What caused you to decide that this should be hardcoded? I think it's probably fine like this, but I'm still curious.

Reply:
@theKunte @nataliagourova @jak-ashuraliev => we probably need to make sure we have identical variable names for other functions as well.

Reply:
I didn't know what to do with the driver so I hardcoded it for now, but I think it belongs to the environment variables. Does anyone have any advice or thoughts?

Reply:
Same thing here, I hardcoded the driver info since that stuff is pulled from the config file found on the cloud. I am sure we can write a script that could pull info from that file and paste into this code

Reply:
Same here. I need more research and see how we can move the driver.

Reply:
We can address this in the next sprint?

Reply:
Yes, I think it's a good task for the next sprint